docs/DeveloperGuide.md

```diff
@@ -383,16 +370,14 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 **MSS**
 
-1.  Actor searches OS directory for save file.
-2.  OS returns save file object.
-3.  For each line of save file.
+1.  Actor retrieve save file object.
+2.  For each line of save file.
 
-    - 3.1 Actor reads a line from the file.
-    - 3.2 Actor parses the line into a contact object.
-    - 3.3 Actor adds the contact object to the contact list.
-    - 3.4 Repeat steps 3.1 - 3.3 until all lines are processed.
+    - 2.1 Actor reads a line from the file.
```

Comment:
Should this be System instead of Actor? since we define Actor as User

Reply:
I reread the definition on Actors and it states "...Actors are not part of the system; they reside outside the system.", and thus removed the UCs.